backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1651,14 +1651,88 @@ databaseChangeLog:
                   constraints:
                     nullable: false
   - changeSet:
-      id: add-person-preferred-language
-      author: nicholas.a.scialli@omb.eop.gov
-      comment: Add column to Person for preferred language
+      id: add-patient-preferences
+      author: adam@skylight.digital
+      comment: Add table for non-demographic preferences relating to a Person
       changes:
-        - addColumn:
-            tableName: person
+        - createTable:
+            tableName: patient_preferences
             columns:
+              - column:
+                  name: internal_id
+                  type: *idtype
+                  remarks: The internal database identifier for this entity.
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: person_internal_id
+                  type: *idtype
+                  remarks: The internal database identifier for this entity.
+                  constraints:
+                    foreignKeyName: fk__patient_preferences__person
+                    references: person
+                    unique: true
+                    nullable: false
               - column:
                   name: preferred_language
                   type: *string
                   remarks: Tracks the individual's preferred language
+              - column:
+                  name: test_result_delivery
+                  type: ${database.defaultSchemaName}.TEST_RESULT_DELIVERY
+                  remarks: Tracks the individual's preference for receiving test results
+              - column:
```

Comment:
Use the aliases for these unless we can't (in which case we should do some upstream clean-up to get our non-scalar alias count down so that new code can be clean).

Reply:
Thank you! Done